jobApplicationTracker/DataModels/JobApplicationHistory.cs

```diff
@@ -1,17 +1,16 @@
 ﻿using System.ComponentModel.DataAnnotations;
 
-namespace jobApplicationTrackerApi.Models
+namespace jobApplicationTrackerApi.DataModels
 {
     public class JobApplicationHistory
     {
         public Guid Id { get; set; }
         public Guid JobApplicationId { get; set; }
         public DateTime UpdatedAt { get; set; } = DateTime.Now;
 
+        public Guid StatusId { get; set; }
```

Comment:
I think we don't need this statusId and Status object since we can retrieve them from the JobApplication model which is connected to status table

Reply:
Thanks for the suggestion @IvanAndrau! The reason we included the StatusId and Status properties in JobApplicationHistory is to maintain an accurate historical snapshot. While we can indeed derive the current status from the JobApplication model, the goal of a history record is to capture the state of that application at a specific point in time. If we rely solely on the JobApplication entity for status, we may lose the historical context if the status changes later. Keeping StatusId in the history model ensures we preserve the status as it was when the history entry was recorded, which can be important for reporting, and understanding the evolution of the job application over time. Let me know what you think!